Library/Homebrew/extend/os/linux/dependency_collector.rb

```diff
@@ -0,0 +1,5 @@
+class DependencyCollector
+  def ant_dep(spec, tags)
+    Dependency.new(spec.to_s, tags)
+  end
```

Comment:
No worries!

That last comment is a helpful clarification for the future, to be sure.